src/components/events-table.js

```diff
@@ -94,7 +94,26 @@ const EventsTable = ({ taskData }) => {
         setLogFilter(filter);
     };
 
+    const isCurrentLogFilterAddErrors = (currentLogFilter) => {
+        let result = false;
+        eventPredefinedFilter.forEach((f) => {
```

Comment:
In the gridcapa-deployment project sometimes we have the variable eventPredefinedFilter which is an array and sometimes it is an object. This causes problems in the front because if we try to do a foreach on an object, it is not allowed.

eventPredefinedFilter defined as an array

![CSE-IMPORT](https://user-images.githubusercontent.com/116628858/226932003-3219ab0f-9a20-41a4-a370-ab3725340742.png)

eventPredefinedFilter defined as an object

![CSE-VALID](https://user-images.githubusercontent.com/116628858/226931995-6ac807a4-71f6-4600-aaa6-2232025fafaa.png)

Problems in the events tab

![Capture d’écran de 2023-03-22 15-00-03](https://user-images.githubusercontent.com/116628858/226932990-2090c4e3-c20f-4e08-8837-8854bfae3479.png)

A basic solution is to put the setEventPredefinedFilter in the if statement but it is preferable that the eventPredefinedFilter is always an array, and remove the if statement.

![Capture d’écran de 2023-03-22 15-26-51](https://user-images.githubusercontent.com/116628858/226935601-e7194ef3-7a51-4d93-93fb-ba98428eac32.png)

Reply:
update gridcapa-deployment so all configurations make "eventPredefinedFilter" an array and not an object :)